source/library/init-player.js

```diff
@@ -58,6 +58,10 @@ export default function initPlayer(element, keyframes, options, render, window =
 		};
 	}
 
+	if (isNaN(options.delay) || isNaN(options.duration) || isNaN(options.iterations)) {
+		return;
```

Comment:
Please log a warning about the non-number value here if `process.env.NODE_ENV === 'development'`